src/util.c

```diff
@@ -66,6 +66,12 @@ int git_strarray_copy(git_strarray *tgt, const git_strarray *src)
 
 int git__strtol64(int64_t *result, const char *nptr, const char **endptr, int base)
 {
+
+	return git__strtonl64(result, nptr, (size_t)-1, endptr, base);
+}
+
+int git__strtonl64(int64_t *result, const char *nptr, size_t nptr_len, const char **endptr, int base)
```

Comment:
Since the length refers to the input string, would this follow the convention more closely if it were `p_strntol64()`? And the same for the 32-bit version?